release-team/role-handbooks/release-signal/README.md

```diff
@@ -0,0 +1,562 @@
+    - Understand and fix the test for the current release
+    - Understand if the test needs to be on the release blocking dashboard
+    - Work with SIG-Testing on any possible test infra improvement to help improve the test pass rate
+- Making recommendations to SIG-Release for promoting and demoting release blocking and merge blocking tests as per
+  the [Blocking Jobs Criteria](/release-blocking-jobs.md)
+- Listing open issues and PRs targeted for the ongoing release cycle
+- Communicating with relevant assignees, owners, and SIG leads of issues/PRs to get the status
+- Updating issues/PRs, clarifying situations, enabling next level decision making
+- Presenting summary reports at release team and burndown meetings
+- Maintaining the automation around tracking issues/PRs against the current milestone
+
+The core responsibility of the Release Signal lead is to foster a culture of continuous test integration, fail-fast and fix-fast
+mindset and strive for continuously passing tests that provide an accurate positive signal to the release team to ensure that
+releases go out on time with high quality. It is critical to maintain a sustained focus on test health through the entire release
+cycle, as opposed to accepting destabilizing test failures during active enhancement development followed by an extended test
+stabilization phase.
+
+## Requirements
+
+### Time Requirements
+
+Release Signal is one of the most time-intensive roles on the release team. As such, it is not recommended for candidates who do
+not have employer support for their work on the Release Team. General time requirements:
+
+- 1/2 hour to 2 hours daily, every workday, checking and following up on failing/flaky tests, issues and PRs targeted for the
+  ongoing release cycle around week 6 (mid-release).
+- Availability to attend most of Release Team (weekly) and Burndown meetings (daily during Code Freeze), subject to time zone
+  appropriateness.
+- Ability to follow up on test fixes during Code Freeze at arbitrary times to ensure rapid turnaround.
+- The time commitment increases through the release cycle, peaking during Code Freeze. In the last week of Code Freeze, shadows
+  should expect to spend at least 10 hours and lead at least 20 hours per week.
+
+### Additional Requirements for Shadows
+
+The following requirements are all things that the lead should already have, and shadows should acquire, with coaching, within the
+first weeks of the release cycle:
+
+- Have signed the contributor CLA for Kubernetes.
+- Become a Kubernetes org member. In many cases, this will be done with the sponsorship of the Release Signal Lead or Release Lead
+  in the first week of the cycle.
+    - The process to become an org member is
+      in [our community membership ladder](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-for-outside-collaborators)
```

Comment:
can we link to another document. These are general requirements across the team, which we don't want to repeat in every handbook.

Reply:
Linked to: https://github.com/kubernetes/sig-release/blob/master/release-team/release-team-onboarding.md#kubernetes-community-membership

Done.